test/test.dart

```diff
@@ -189,7 +189,11 @@ Future<void> setUpAppTest([VoidCallback? configureFakes]) async {
   ThemeControl.instance.init();
   ThemeControl.instance.initSystemUi();
   await Permissions.instance.init();
-  await ContentControl.instance.init();
+  if (binding.inTest) {
+    await binding.runAsync(() => ContentControl.instance.init());
```

Comment:
Because we are doing real async work with the `await Future<void>.delayed(const Duration(milliseconds: 100));` in `test/fakes/fake_just_audio.dart:147`

This caused the test to hang (see [here](https://github.com/nt4f04uNd/sweyer/actions/runs/4644452852/jobs/8219736699#step:3:875)).

Originally, I wanted to is `binding.delayed` instead of `Future.delayed`, but because this is apparently executed in the background while the test is running, it threw an error about a reentrant use of the delayed function.

Reply:
It appears to me it timed out there, because you didn't update this specific permission test. But I think I'm okay with this change overall.